recommenders/evaluation/spark_evaluation.py

```diff
@@ -560,6 +565,21 @@ def __init__(
             self.reco_df = reco_df.select(
                 col_user, col_item, F.col(self.col_relevance).cast(DoubleType())
             )
+            
+        if item_feature_df is None:
+            self.item_feature_df = None
+        else:
+            required_schema = StructType(
+               (
+                 StructField(self.col_item, IntegerType()),
+                 StructField("features", VectorUDT()),
```

Comment:
can we create self.col_features so the user can control the column name just like user, item, relevance

Reply:
+1

Reply:
addressed